spec/CloudCode.spec.js

```diff
@@ -39,6 +40,34 @@ describe('Cloud Code', () => {
     });
   });
 
+  it('should wait for cloud code to load', async () => {
+    const config = Config.get('test');
+    const initiated = new Date();
+    const parseServer = await ParseServer.createAsync({
+      ...config,
+      async cloud() {
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Add a new test for "startAsync" to cover that we still support `startServerComplete` and we do not overwrite the function internally.